schaetzmeisterin-backend-server/pom.xml

```diff
@@ -24,6 +24,12 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
+		<!-- Test -->
+		<dependency>
```

Comment:
i think this is already build in spring test

Reply:
Thx, works without the dependency